src/lib/appwrite/services/auth.service.ts

```diff
@@ -31,8 +31,9 @@ export const authService = {
   },
   async createUserVerification() {
     try {
-      // TODO: This url will be replaced after that route is created in frontend.  
-      return await account.createVerification("url");  
+      return await account.createVerification(
+        "http://localhost:5173/confirm-email"
```

Comment:
dont put hardcoded string instead use env variables. put it inside your .env file as VITE_CONFIRM_EMAIL_URL=http://localhost:5173/confirm-email